src/test/java/guitests/KeyboardShortcutsTest.java

```diff
@@ -171,6 +171,7 @@ public void keyboardShortcutsTest() {
         assertEquals(repoSelectorComboBox.getValue(), "dummy/dummy");
         // test shortcut when focus is on panel
         PlatformEx.waitOnFxThread();
+        sleep(1000);
```

Comment:
The problem here is not col1 but rather dummy/dummy. The repo change makes the node traversal very unpredictable. I tried dummy/dummy_blah and dummy1/dummy1_blah and both gives Node not found. I can make it wait until dummy/dummy_col1 can be found, but it will make the time non-deterministic. Would that be ok?